README.md

```diff
@@ -55,21 +62,21 @@ All options are optional.
 For example, here is how you would add a request count metric using `uber-statsd-client`:
 
 ```javascript
-var app = require('cadence-web')
```

Comment:
Our code is configured to use single quote so the existing format matches the project guidelines

Reply:
Ahh, okay
Missed this comment - it should be resolved now